addons/html_editor/static/tests/editor/toolbar.test.js

```diff
@@ -60,3 +61,22 @@ test("toolbar buttons react to selection change", async () => {
     await waitFor(".btn[name='bold'].active");
     expect(".btn[name='bold']").toHaveClass("active");
 });
+
+test("toolbar buttons react to selection change (2)", async () => {
+    const { el } = await setupEditor("<p><strong>test [some]</strong> some text</p>");
+
+    await waitFor(".o-we-toolbar");
+
+    // check that bold button is not active
```

Comment:
~~not~~ active!

Reply:
oups. since the comment is not really useful, i just removed it